tests/test_transformations.py

```diff
@@ -65,7 +65,7 @@ class Myschema(Schema):
         m2 = Myschema()
 
         self.assertEqual(first, All()(m).serialize())
-        self.assertEqual({}, All()(m2).serialize())
+        self.assertEqual({}, All()(m2).serialize(implicit_nulls=True))
```

Comment:
this is one where the test is more meaningful if you change the asserted value and let implicit_nulls=False (implicitly)